### PR TITLE
feat: script to update expired listings

### PR DESCRIPTION
**Pull request by schweinryder**

### CODE REVIEW

File: src/services/lease-service/adapters/listing-adapter.ts [hunk 1]
```diff
@@ -300,6 +300,23 @@ const applicationExists = async (contactCode: string, listingId: number) => {
   return !!result // Convert result to boolean: true if exists, false if not
 }
 
+const getExpiredListings = async () => {
+  const currentDate = new Date()
+  const listings = await db('listing')
+    .where('PublishedTo', '<', currentDate)
+    .andWhere('Status', '!=', 4) // Assuming 4 is the status code for 'Expired'
```
Comment (driatic) [inline src/services/lease-service/adapters/listing-adapter.ts:307]: use `ListingStatus.Expired`?

**Review by driatic — COMMENTED** [review]
How is this supposed to run in the kubernetes context? How would the scheduling work? I assumed a separate node app (as in an index file that runs `updateExpiredListings()` and a docker file that spins up that app in the kubernetes cluster. 

The final outcome of the feature should most likely be a cronjob in kubernetes: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

File: src/services/lease-service/adapters/listing-adapter.ts (hunk 1 above)
Comment (schweinryder) [reply src/services/lease-service/adapters/listing-adapter.ts]: It's not in yet :)

**Comment by schweinryder** [comment]
> How is this supposed to run in the kubernetes context? How would the scheduling work? I assumed a separate node app (as in an index file that runs `updateExpiredListings()` and a docker file that spins up that app in the kubernetes cluster.
> 
> The final outcome of the feature should most likely be a cronjob in kubernetes: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

See the PR in the operations repo

**Pull request merged**